### DISCUSSION TITLE
No Access to GitHub Copilot found

### DISCUSSION THREAD

**Discussion by tea-scripts** [community/community#44695] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hello, I can't seem to find what went wrong. I'm using the student pack so my github account is a PRO account and then I have been using copilot since July 2022 but suddenly today it stopped working.

[Error Log screenshot](https://res.cloudinary.com/teascript/image/upload/v1674063659/Screenshot_2023-01-19_012813_gnhj6a.png)

I have tried:

- Reinstalling the extension
- Signing out my github account from vs code and signing in (multiple times)
- Signing our of github from the web browser and signing in (got this from another article)
- Restarting my PC.

**Comment by loukai99** [comment]
i have same problem

**Comment by lemmbellaht** [comment]
me too

**Comment by ssakib4040** [comment]
same here
